olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -19,16 +19,16 @@
 
 #include "olp/authentication/AuthenticationClient.h"
 
-#include <chrono>
-#include <sstream>
-
 #include <rapidjson/document.h>
 #include <rapidjson/istreamwrapper.h>
 #include <rapidjson/stringbuffer.h>
 #include <rapidjson/writer.h>
+
 #include <boost/uuid/uuid.hpp>
 #include <boost/uuid/uuid_generators.hpp>
 #include <boost/uuid/uuid_io.hpp>
+#include <chrono>
+#include <sstream>
```

Comment:
Boost is not a system library.
Please move system includes back to where they were before.